ApollosProject/apollos-apps
# fix: Ensure we don't try and zoom the map to a remote campus

## DESCRIPTION

### What does this PR do, or why is it needed?

When I refactored this line of code in a recent commit (https://github.com/ApollosProject/apollos-apps/commit/cd0ef4db323bf234ba952aaee65b94ece45369f0) I refactored it wrong. We want to filter out both the `currentLocation` and the `sortedCampuses` array, instead of just the `sortedCampuses` array. 

![2020-10-01 14 36 13](https://user-images.githubusercontent.com/1637694/94859921-5147b900-0403-11eb-8b1e-491a27845d59.gif)


### What design trade-offs/decisions were made?

### How do I test this PR?

Open the `ConnectedMapView` story and try to scroll to the Online Campus. The app should not crash.

### What automated tests did you write?

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
